packages/flutter/lib/src/widgets/scroll_physics.dart

```diff
@@ -429,7 +429,11 @@ class ScrollPhysics {
 /// contents change dimensions suddenly.
 ///
 /// If the scroll position is already out of range, this attempts to maintain
-/// the amount of overscroll or underscroll already present.
+/// the amount of overscroll or underscroll already present _and_ the position
+/// has shrunk, meaning that some content was removed. The reason for this
```

Comment:
i'm not sure i understand this first sentence any more, can you see if it's what you intended for it to say?

Reply:
Yeah, this was indeed an invalid sentence. Changed it